frontend/__tests__/page.test.tsx

```diff
@@ -25,4 +47,14 @@ describe('Home', () => {
   it('renders the card screen', () => {
     expect(screen.getByTestId('card-screen')).toBeVisible()
   })
+
+  describe('when accept button is clicked',  () => {
+    it('the restaurant is saved to the like list', async () => {
+      await userEvent.click(screen.getByRole('button', { name: 'Accept' }))
+
+      const matchList = screen.getByTestId('match-list')
```

Comment:
Also, related to the same guiding principles mentioned earlier, we generally should also avoid using `data-testid`s when querying for an element as a user would never see or use test ids.  

From the [docs on byTestId](https://testing-library.com/docs/queries/bytestid/):

> In the spirit of [the guiding principles](https://testing-library.com/docs/guiding-principles), it is recommended to use this _only after_ the other queries don't work for your use case. **Using data-testid attributes do not resemble how your software is used and should be avoided if possible.** That said, they are way better than querying based on DOM structure or styling css class names. Learn more about data-testids from the blog post ["Making your UI tests resilient to change"](https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change)

In your case, I wonder if we can use something else... Although I do see the reason why you chose to use testId here. 🤔 (I'm going to come back to this comment since I have to go right now)

Reply:
Didn't have the time to think about alternatives to this. Going to approve the PR and maybe we can come back to this concept at a later point. :)